101-app-function/azuredeploy.json

```diff
@@ -0,0 +1,66 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "siteName": {
+            "type": "string",
+            "defaultValue": "[concat('FuncApp-', uniqueString(resourceGroup().name))]",
+            "metadata": {
+                "description": "The name of you Web Site."
+            }
+        },
+        "location": {
+        "type": "string",
+        "defaultValue": "[resourceGroup().location]",
+        "metadata": {
+                "description": "Location for all resources."
+            }
+        }
+    },
+     "variables": {
+        "hostingPlanName": "[concat('hpn-', uniqueString(resourceGroup().id))]"
```

Comment:
use resourceGroup().name as in your other samples, e.g.

concat('hpn-', resourceGroup().name)

Reply:
Also done!